spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1433,6 +1434,45 @@ static void queryWithTimestampParameter(DatabaseClient dbClient) {
   }
   // [END spanner_query_with_timestamp_parameter]
 
+  // [START spanner_create_client_with_query_options]
+  static void clientWithQueryOptions(DatabaseId db) {
+    SpannerOptions options =
+        SpannerOptions.newBuilder()
+            .setDefaultQueryOptions(
+                db, QueryOptions.newBuilder().setOptimizerVersion("latest").build())
+            .build();
+    Spanner spanner = options.getService();
+    DatabaseClient dbClient = spanner.getDatabaseClient(db);
+    try (ResultSet resultSet =
+        dbClient
+            .singleUse()
+            .executeQuery(Statement.of("SELECT SingerId, AlbumId, AlbumTitle FROM Albums"))) {
+      while (resultSet.next()) {
+        System.out.printf(
+            "%d %d %s\n", resultSet.getLong(0), resultSet.getLong(1), resultSet.getString(2));
+      }
+    }
+  }
+  // [END spanner_create_client_with_query_options]
+
+  // [START spanner_query_with_query_options]
+  static void queryWithQueryOptions(DatabaseClient dbClient) {
+    try (ResultSet resultSet =
+        dbClient
+            .singleUse()
+            .executeQuery(
+                Statement
+                    .newBuilder("SELECT SingerId, AlbumId, AlbumTitle FROM Albums")
+                    .withQueryOptions(QueryOptions.newBuilder().setOptimizerVersion("latest").build())
```

Comment:
Same here, I think it's useful to show how to set the version to something other than the default.

```suggestion
                    .withQueryOptions(QueryOptions.newBuilder().setOptimizerVersion("1").build())
```

Reply:
Done.